feature/bgp/policybase/otg_tests/import_export_multi/README.md

```diff
@@ -0,0 +1,232 @@
+    * MED should be not present
+
+## Procedure
+
+### RT-7.11.1 - Create a bgp policy containing the following conditions and actions
+
+* Summary of this policy
+  * Reject route matching any communities in a community-set.
+  * Reject route matching another policy (nested) and not matching a community-set.
+  * Add a community-set if missing that same community-set.
+  * Add two communities and set localpref if matching a community and prefix-set.
+  * Set MED if matching an aspath
+
+* Define a policy that will be called from another policy
+  * policy-definitions/policy-definition/config/name: "match_community_regex"
+    * statements/statement/config/name: "match_community_regex"
+      * conditions/bgp-conditions/match-community-set/config/
+        * community-set: "regex-community"
+        * match-set-options: "ANY"
+      * actions/config/policy-result = "NEXT_STATEMENT"
+
+* Create policy-definitions/policy-definition/config/name = "multi_policy"
+  * statements/statement/config/name = "reject_route_community"
+    * conditions/bgp-conditions/match-community-set/config
+      * community-set = "reject_communities"
+      * match-set-options = "ANY"
+    * actions/config/policy-result = "REJECT_ROUTE"
+
+  * statements/statement/config/name = "if_30:.*_and_not_20:1_nested_reject"
+    * conditions/config/call-policy = "match_community_regex"
+    * conditions/bgp-conditions/match-community-set/config/
+      * community-set = "accept_communities"
+      * match-set-options = "INVERT"
+    * actions/config/policy-result = "REJECT_ROUTE"
+
+  * statements/statement/config/name = "add_communities_if_missing"
+    * conditions/bgp-conditions/match-community-set/config/
+      * community-set-refs = "add-communities"
+      * match-set-options: "INVERT"
+    * actions/bgp-actions/set-community/reference/config/
+      * community-set-refs = "add-communities"
+      * method = "REFERENCE"
+      * option = "ADD"
+    * actions/config/policy-result = "NEXT_STATEMENT"
+
+  * statements/statement/config/name: "match_comm_and_prefix_add_2_community_sets"
+    * conditions/bgp-conditions/match-community-set/config
+      * community-set = "my_community"
+      * match-set-options = "ANY"
+    * conditions/match-prefix-set/config
+      * prefix-set = "prefix-set-5"
+      * match-set-options = "ANY"
+    * actions/bgp-actions/set-community/config
+      * method = "REFERENCE"
+      * option = "ADD"
+      * community-set-refs = "add_comm_60", "add_comm_70"
+    * actions/bgp-actions/config/set-local-pref = 5
+    * actions/config/policy-result = "NEXT_STATEMENT"
+
+  * statements/statement/config/name: "match_aspath_set_med"
+    * conditions/bgp-conditions/match-as-path-set/config/
+      * as-path-set = "my_aspath"
+      * match-set-options = "ANY"
+    * actions/bgp-actions/config/
+      * set-med = 100
+    * actions/config/policy-result = "ACCEPT_ROUTE"
+
+#### RT-7.11.1.1 Configure import_multi_policy
```

Comment:
```suggestion
#### RT-7.11.2.1 attach multi_policy as import policy
```

Reply:
updating the number to break out these steps